### PR TITLE
Indexation augmentation (forgot this wasnt in here)

### PR DESCRIPTION
**Pull request by ctr26**
This is actually two fixes

- The one is that the normalising to window size logic was wrong (there has to be a window size in the current code even if that window size if the whole image it just shouldnt be pixel distances)
- I forgot to add the indexation augmentation code before, this should give the model significantly more augmented data to work with, i.e. 256 times which will eventually help it learn.
-- Unsure whether the amount of augmentation should be in the dataloader or in the transform, transform is easier

### CODE REVIEW

File: bioimage_embed/shapes/transforms.py [hunk 1]
```diff
@@ -420,3 +418,20 @@ def asym_dist_to_sym_dist(self, asymm_dist):
 
         sym_dist = np.max(dist_stack, axis=0)
         return torch.tensor(np.array(sym_dist))
+
+
+class RotateIndexingClockwise(nn.Module):
```
Comment (ctr26) [inline bioimage_embed/shapes/transforms.py:423]: @afoix Added here

**Review by afoix — APPROVED** [review]
I tested this change with the full pipeline and it's working so, should be merge in master and PR close.
